omod/src/main/java/org/openmrs/module/kenyaemr/fragment/controller/RegimenUtilFragmentController.java

```diff
@@ -54,7 +57,7 @@
 public class RegimenUtilFragmentController {
 
 	protected static final Log log = LogFactory.getLog(RegimenUtilFragmentController.class);
-
+	private Date enrollmentDate = null;
```

Comment:
@makombe kindly refer to DateOfEnrollmentArtCalculation for the best approach to use for this. One is likely to have multiple enrollments and you should consider the very first enrollment. That should give you the answers.